transform/company.go

```diff
@@ -104,13 +103,13 @@ func (c *company) situacaoCadastral(v string) error {
 	return nil
 }
 
-func (c *company) cnaeSecundarios(v string) error {
+func (c *company) cnaeSecundarios(l lookups, v string) error {
 	for _, n := range strings.Split(v, ",") {
-		i, err := strconv.Atoi(n)
+		cnae, err := getCnaeInfo(l, n)
 		if err != nil {
-			return fmt.Errorf("error converting %s to int: %w", v, err)
+			return fmt.Errorf("error getting Cnae description %w", err)
```

Comment:
Sempre vejo erros com tudo em minúsculo, então tento manter esse padrão tb:

```suggestion
			return fmt.Errorf("error getting cnae description %w", err)
```